server/message_hooks.go

```diff
@@ -549,6 +549,8 @@ func (p *Plugin) SendChat(srcUser string, usersIDs []string, post *model.Post) (
 		return "", err
 	}
 
+	timeElapsed := float64((newMessage.CreateAt.UnixMilli() - post.CreateAt)) / float64(1000)
```

Comment:
This is useful too, but it is again generating an unnecesary uncontrolled information. If I deliver the message to MSTeams, I don't need to use the date from MSTeams, I can use my date, using the ellapsed time.